test/libraries/FullMath.t.sol

```diff
@@ -205,4 +206,40 @@ contract FullMathTest is Test {
 
         return mulDivResultOverflows || mulDivRoundingUpResultOverflows;
     }
+
+    function test_mulDiv_matchesJavaScriptImpl(uint256 a, uint256 b, uint256 denominator) public {
```

Comment:
anything that uses ffi let's put in the test name so we can filter out these tests if we need to when we run locally

`test_ffi_fuzz_...`